app/core/store/modules/me.js

```diff
@@ -60,6 +64,26 @@ export default {
 
     isSmokeTestUser (state) {
       return utils.isSmokeTestEmail(state.email)
+    },
+
+    hasSubscription (state) {
```

Comment:
This is an excellent refactor from https://github.com/codecombat/codecombat/blob/0c14ec0d177ae65555df7a882d56dac2ab3c586e/app/models/User.coffee#L302

Interested in your thoughts regarding reusing this logic somehow, or how we do this migration. This store is very specific to the `me` user, but is reusing model specific behavior. Maybe the solution is to make our models less tied to data. Random thoughts and no action required.

Reply:
All good questions, I moved this here in the spirit of moving toward Vuex and considered calling the User model directly as a first migration step but decided against it as the implementation of hasSubscription seems to have a dependency on window.me which I think is a bug (or some kind of access control implementation).  Regardless, I thought an implementation in Vuex would be cleaner.  That said the implementation is specific to the current user object and we may want to use it elsewhere.  It'd likely make sense to extract that into a shared method somewhere when we need it.